package.json

```diff
@@ -10,6 +10,7 @@
     "screener-runner": "./bin/screener-runner.js"
   },
   "scripts": {
+    "postinstall": "SAUCE_CONNECT_DOWNLOAD_ON_INSTALL=true SAUCE_CONNECT_VERSION=4.5.4 node ./node_modules/sauce-connect-launcher/scripts/install.js",
```

Comment:
@xwa130saucer check the original post-install script; https://github.com/bermi/sauce-connect-launcher/blob/master/package.json#L21
This is probably done this way to support different node versions, we should try to handle this too.

Reply:
@xwa130saucer what do you think of this approach as an alternative:
```
SAUCE_CONNECT_DOWNLOAD_ON_INSTALL=true SAUCE_CONNECT_VERSION=4.5.4 npm run postinstall --prefix ./node_modules/sauce-connect-launcher
```

Reply:
> @xwa130saucer check the original post-install script; https://github.com/bermi/sauce-connect-launcher/blob/master/package.json#L21
> This is probably done this way to support different node versions, we should try to handle this too.

From my understanding, the later part `|| nodejs scripts/install.js` is a guarantee that the system does not have `node` instead of `nodejs` command line. But I don't understand how they handle version... I'll add that part in case of the host does not have `node` command line. And could you please clarify the version handle part? didn't quite get it. 


> @xwa130saucer what do you think of this approach as an alternative:
> 
> ```
> SAUCE_CONNECT_DOWNLOAD_ON_INSTALL=true SAUCE_CONNECT_VERSION=4.5.4 npm run postinstall --prefix ./node_modules/sauce-connect-launcher
> ```

I think this command line will fire the `sauce-connect-launcher` which is not what we want to do right? We are just downloading the sauce connect client at this phase. 
Also, from my understanding, the postinsall is a hook that is automatically called when the package is being installed.

Reply:
@xwa130saucer `npm run postinstall --prefix` will launch the post-install script, not the sauce-connect-launcher. I can show you this on zoom tomorrow.

Reply:
@vrunoa, Sorry about the late reply, this somehow fell through the crack... 
Let's talk in person when I come back to our office about this : )